frontend/src/pages/Admin/AdminArtistPage/AdminArtistPage.module.css

```
@@ -0,0 +1,87 @@
+  color: #f5c518;
+}
+
+.tableOuter {
+  background: #1f1f1f;
+  border-radius: 12px;
+  padding: 0.5rem;
+  border: 1px solid #333;
+  box-shadow: 0 20px 40px rgba(0, 0, 0, 0.5);
+}
+
+.tableOuter :global(thead th) {
```

Stylelint (17.5.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)